runtime/src/lib.rs

```diff
@@ -60,21 +59,21 @@ pub const STABLE_PARACHAIN_CONFIRMATIONS: &str = "StableParachainConfirmations";
     feature = "parachain-metadata-kintsugi",
     subxt(
         runtime_metadata_path = "metadata-parachain-kintsugi.scale",
-        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+        generated_type_derives = "Eq, PartialEq, Ord, PartialOrd, Clone"
     )
 )]
 #[cfg_attr(
     feature = "parachain-metadata-testnet",
     subxt(
         runtime_metadata_path = "metadata-parachain-testnet.scale",
-        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+        generated_type_derives = "Eq, PartialEq, Ord, PartialOrd, Clone"
     )
 )]
 #[cfg_attr(
     feature = "standalone-metadata",
     subxt(
         runtime_metadata_path = "metadata-standalone.scale",
-        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+        generated_type_derives = "Eq, PartialEq, Ord, PartialOrd, Clone"
```

Comment:
Why not derive debug? That can be quite useful in debugging

Reply:
Already added by subxt: https://github.com/paritytech/subxt/blob/a7c54f74127b58dd0886b9f42b1f7318dfbc51f4/codegen/src/types/derives.rs#L53